app/Models/Photo.php

```diff
@@ -336,7 +336,7 @@ protected function getLivePhotoUrlAttribute(): ?string
 	protected function getIsDownloadableAttribute(): bool
 	{
 		return
-			AccessControl::is_current_user_or_admin($this->owner_id) ||
+			Gate::check('own', $this) ||
 			($this->album_id !== null && $this->album->is_downloadable) ||
 			($this->album_id === null && Configs::getValueAsBool('downloadable'));
 	}
```

Comment:
I disagree with `owner-or-admin` instead of `own`, because you can just define admin in the `before()` part of the gate and thus removing this implication check.

Reply:
Yes, I withdraw that comment. This only became clear to me afterwards, because I reviewed the source file for the Gate later after I already had written this comment.

This comment can be resolved.